staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -682,6 +682,8 @@ message WindowsContainerResources {
     int64 cpu_maximum = 3;
     // Memory limit in bytes. Default: 0 (not specified).
     int64 memory_limit_in_bytes = 4;
+    // Amount of CPUs (e.g. 2000000000 = 2 CPU cores)
+    int64 nano_cpus = 5;
```

Comment:
I managed to avoid modifying CRI in https://github.com/kubernetes/kubernetes/pull/86101, but am still testing it. Can you review the approach there and let me know what you think?

Reply:
Already replay :)